generator/internal/discovery_type_vertex.cc

```diff
@@ -386,6 +359,38 @@ DiscoveryTypeVertex::FormatProperties(  // NOLINT(misc-no-recursion)
   return message_properties;
 }
 
+std::string DiscoveryTypeVertex::FormatMessageDescription(
+    nlohmann::json const& field, int indent_level) {
+  std::string description;
+  if (field.contains("description")) {
+    auto sanitized_description = std::string(field["description"]);
+    // If there are any $ in the description, we want to replace it with two $$
```

Comment:
```suggestion
// Replace the $ in the description with $$ so the protoc compiler 
// will emit a $ instead of trying to perform a substitution.
```